pkg/dashboards/types.go

```diff
@@ -271,7 +385,13 @@ type DashboardWidgetVisualizationInput struct {
 	ID string `json:"id,omitempty"`
 }
 
+// EpochMilliseconds - The `EpochMilliseconds` scalar represents the number of milliseconds since the Unix epoch
+type EpochMilliseconds string
```

Comment:
Need to add some sort of package ownership for Types into Tutone so when auto-generation happens, Types owned by another package aren't created yet again.